src/Error/Debug/dumpHeader.html

```diff
@@ -1,275 +1,284 @@
-  })
-  target.dataset.highlighted = 'true';
-
-  var current = target;
-  // Traverse up the tree opening all closed containers.
-  while (true) {
-    var parent = current.parentNode;
-    if (parent == container) {
-      break;
+    .cake-debug-property {
+        color: var(--color-dark-grey);
     }
-    if (parent.classList.contains('cake-dbg-object') || parent.classList.contains('cake-dbg-array')) {
-      expander = parent.querySelector('.cake-dbg-collapse');
-      if (expander.dataset.open === 'false') {
-        expander.click();
-      }
+
+    .cake-debug-visibility {
+        color: var(--color-violet);
     }
-    current = parent;
-  }
-}
-
-// https://www.30secondsofcode.org/js/s/copy-to-clipboard
-function copyToClipboard (str) {
-  var el = document.createElement('textarea');
-  el.value = str;
-  el.setAttribute('readonly', '');
-  el.style.position = 'absolute';
-  el.style.left = '-9999px';
-  document.body.appendChild(el);
-  var selected = document.getSelection().rangeCount > 0 ? document.getSelection().getRangeAt(0) : false;
-  el.select();
-  document.execCommand('copy');
-  document.body.removeChild(el);
-  if (selected) {
-    document.getSelection().removeAllRanges();
-    document.getSelection().addRange(selected);
-  }
-}
-
-function attachCopyButton(nodes) {
-  nodes.forEach(function (container) {
-    var copy = doc.createElement('a');
-    copy.classList.add('cake-dbg-copy');
-    copy.setAttribute('href', '#');
-    copy.setAttribute('title', 'Copy contents of debug output');
-    copy.appendChild(doc.createTextNode('Copy'));
-
-    // Add copy behavior
-    copy.addEventListener('click', function (event) {
-      event.preventDefault();
-      event.stopPropagation();
-      var lineNo = '';
-      if (container.parentNode && container.parentNode.classList.contains('cake-debug')) {
-        var line = container.parentNode.querySelector('span');
-        lineNo = line.textContent + "\n";
-      }
-
-      // Chop off last 4 to exclude copy button text.
-      copyToClipboard(lineNo + container.textContent.substring(0, container.textContent.length - 4));
-    });
-
-    container.appendChild(copy);
-  });
-}
-
-doc.addEventListener('DOMContentLoaded', initialize);
-}(window, document))
+
+    .cake-debug-punct {
+        color: var(--color-grey);
+    }
+
+    .cake-debug-string {
+        color: var(--color-green);
+        white-space: pre-wrap;
+    }
+
+    .cake-debug-number {
+        color: var(--color-blue);
+        font-weight: bold;
+    }
+
+    .cake-debug-const {
+        color: var(--color-orange);
+        font-weight: bold;
+    }
+
+    .cake-debug-ref {
+        color: var(--color-red);
+    }
+
+    .cake-debug-special {
+        color: var(--color-red);
+        font-style: italic;
+    }
+</style>
+<script>
+    ( function( win, doc ) {
+
+        function initialize() {
+            createCollapsibles( doc.querySelectorAll( '.cake-debug-array-items' ) );
```

Comment:
Did you use a formatter to add spaces around the method calls? I don't remember these being here before.

Reply:
I did use a formatter but it seems I used the wrong preset. let me revert that.